zapisy/apps/schedule/templates/schedule/event.html

```diff
@@ -125,11 +125,11 @@ <h2>
                 <div class="jumbotron p-2 bg-light border">
                     <div class="row">
                         <div class="col">
-                        <h5 class="display-5">
+                        <h5 class="display-7">
```

Comment:
Zdaje się, że tu (i w jeszcze jednym miejscu w tym szablonie) jedna nieistniejąca (w BS4) klasa została zamieniona na inną nieistniejącą (w BS5). Skoro wszystko było dobrze z domyślnym stylowaniem dla `<h5>` to niech tak zostanie.
```suggestion
                        <h5>
```

Reply:
Zmienione, ale

Patrząc linię poniżej znajdujemy 
```
<h5>
       <a href="">{{ message.author.get_full_name }}</a>
</h5>
```
Czy ten element na pewno powinien być linkiem?

Reply:
Świetna uwaga! Generalnie mógłby, ale na pewno nie do tej samej strony, tylko ewentualnie do profilu danej osoby. Żeby nie rozbudowywać sobie dodatkowo pracy w ramach tego PRa i trzymać się jego tematu, proszę usunąć ten link, oraz analogiczny w "innym miejscu tego pliku".

Reply:
Gotowe